tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933160.yaml

```diff
@@ -66,187 +73,200 @@ tests:
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             method: POST
             port: 80
-            uri: /?foo=exec%0A%28%27bar%27%29
+            uri: /post
```

Comment:
@theseion Have we lost something here? Or this was moved to line 93?

Reply:
Some of the original tests contained more than one payload. @theMiddleBlue cleaned that up. In this case, the test for this line is indeed further down and the line reappears on 93, as you say.

Reply:
Arg, I think we did the work twice :( https://github.com/coreruleset/coreruleset/pull/3462

Reply:
Oh 😅. Well, then we can compare the two for errors at least. Rebasing wasn't easy.